Libplanet/Store/IStateStore.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Immutable;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Blocks;
+
+namespace Libplanet.Store
+{
+    public interface IStateStore
```

Comment:
- This should be recorded in the list of added APIs of the changelog.
- Could you write an XML comment for this interface? It should explain its purposes.